packages/main/src/index.ts

```diff
@@ -68,16 +77,21 @@ function updateAppAndQuit() {
   if (import.meta.env.PROD) {
     updater.autoUpdater.autoInstallOnAppQuit = true;
     updater.autoUpdater.autoRunAppAfterInstall = false;
+    let version = '';
 
     updater.autoUpdater.on('checking-for-update', () => {
       log.info('[Main Window][AutoUpdater] Checking for updates');
```

Comment:
Can we add tracking here as well to check if this event happens?

Reply:
added!